package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js"
+    "start": "nodemon index.js"
```

Comment:
I forgot to change the script, sorry. I'm kinda rushing when making pull request